kolibri/core/assets/src/views/AppError/ReportErrorModal.vue

```diff
@@ -95,11 +96,3 @@
 
 </script>
 
-
-<style lang="scss" scoped>
-
-  .error-detail-modal {
-    text-align: left;
-  }
```

Comment:
if you're removing an entire class, you should remove the reference to it in the `template` also